extensions/amp-analytics/0.1/requests.js

```diff
@@ -318,6 +345,73 @@ export class RequestHandler {
     }
     return s.join('&');
   }
+
+  /**
+   * Handle batchInterval
+   */
+  initBatchInterval_() {
+    if (!this.batchInterval_) {
+      return;
+    }
+
+    if (this.maxDelay_) {
+      // Ignore maxDelay in presence of batchInterval
```

Comment:
so what's the difference between `maxDelay: 2` and `batchInterval: 2`?

Reply:
Talked offline. Decided to keep both.